__tests__/app.test.js

```diff
@@ -57,7 +57,9 @@ describe("/api/articles/:article_id", () => {
   it("GET responds with status code 200", () => {
     return request(app)
       .get("/api/articles/2")
+      .expect(200)
       .then((response) => {
+        expect(response.body.article[0].article_id).toEqual(2)
```

Comment:
I do feel it is a little lazy to be only checking that the id matches when you'll have access to what the author, title is etc.